src/Applications/LDAS_App/lenkf.j.template

```diff
@@ -68,6 +68,18 @@ else
    exit
 endif
 
+#######################################################
+#  if LADAS_COUPLING ==2, do atmens forcing processing 
+###################################################### 
+   if ( $LADAS_COUPLING == 2) then
+     cd $HOMDIR
+     module load  python/GEOSpyD/Ana2019.03_py3.7
+     module  load nco/4.8.1
+     python  average_ensemble_forcing.py
```

Comment:
Doesn't this script have to be added to the repository?

Reply:
it was added in app/util last month  (the the same time with the merge for ensemble increments).